src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/privateApi/Tags.kt

```diff
@@ -1,4 +1,4 @@
-package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.resource
```

Comment:
This Tags and ControllerAnnotations could sit at the higher level as they would potentially be shared by both private and public apis